README.md

```diff
@@ -21,23 +21,28 @@ dokku plugin:install https://github.com/dokku/dokku-rabbitmq.git rabbitmq
 ## commands
 
 ```
-rabbitmq:clone <name> <new-name>  NOT IMPLEMENTED
-rabbitmq:connect <name>           NOT IMPLEMENTED
-rabbitmq:create <name>            Create a rabbitmq service with environment variables
-rabbitmq:destroy <name>           Delete the service and stop its container if there are no links left
-rabbitmq:export <name> > <file>   NOT IMPLEMENTED
-rabbitmq:expose <name> [port]     Expose a rabbitmq service on custom port if provided (random port otherwise)
-rabbitmq:import <name> <file>     NOT IMPLEMENTED
-rabbitmq:info <name>              Print the connection information
-rabbitmq:link <name> <app>        Link the rabbitmq service to the app
-rabbitmq:list                     List all rabbitmq services
-rabbitmq:logs <name> [-t]         Print the most recent log(s) for this service
-rabbitmq:promote <name> <app>     Promote service <name> as RABBITMQ_URL in <app>
-rabbitmq:restart <name>           Graceful shutdown and restart of the rabbitmq service container
-rabbitmq:start <name>             Start a previously stopped rabbitmq service
-rabbitmq:stop <name>              Stop a running rabbitmq service
-rabbitmq:unexpose <name>          Unexpose a previously exposed rabbitmq service
-rabbitmq:unlink <name> <app>      Unlink the rabbitmq service from the app
+rabbitmq:clone <name> <new-name>											NOT IMPLEMENTED
```

Comment:
Why is the diff so large? Did you switch to using tabs instead of spaces? It's hard to see what was added :)